apps/assisted_tagging/schema.py

```diff
@@ -159,14 +140,9 @@ class Meta:
             'threshold',
             'is_selected',
         )
-
-    @staticmethod
-    def resolve_wrong_prediction_reviews(root, info, **kwargs):
-        return root.wrong_prediction_reviews.all()   # NOTE: Prefetched by DraftEntry
-
-    @staticmethod
-    def resolve_model_version_deepl_model_id(root, info, **kwargs):
-        return root.model_version.model.model_id   # NOTE: Prefetched by DraftEntry
+    '''
+    NOTE: model_version_deepl_model_id and wrong_prediction_review is not include here because they are not used in frontend
+    '''
```

Comment:
```suggestion
    '''
    NOTE: model_version_deepl_model_id and wrong_prediction_review are not included here because they are not used in frontend
    '''
```